app.py

```diff
@@ -13,13 +13,16 @@
     Stores,
     ElementsIDs,
     Dimensions,
+    ModelInputs3
```

Comment:
This should not be an input. We need to find a more elegant way to import and define this. Let's have a chat.